okhttp/src/main/java/com/squareup/okhttp/internal/http/CacheStrategy.java

```diff
@@ -67,10 +67,13 @@ public static boolean isCacheable(Response response, Request request) {
       case HTTP_MOVED_TEMP:
       case HTTP_TEMP_REDIRECT:
         // These codes can only be cached with the right response headers.
+        // http://www.w3.org/Protocols/rfc2616/rfc2616-sec13.html#sec13.4
+        // s-maxage is not checked because OkHttp is a private cache that should ignore s-maxage as
+        // per http://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.9.3
```

Comment:
There's new RFCs! The new hotness for caching is here:

http://tools.ietf.org/html/rfc7234
(and friends)